### PR TITLE
Next edition of This Month in Solid

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**

### CODE REVIEW

File: _posts/newsletter/2020-07-02-this-month-in-solid.md [hunk 1]
```diff
@@ -8,3 +8,44 @@ categories: [Updates]
...
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+If you missed Solid World you can [watch the recording here](https://vimeo.com/434767351).
+
+This month, Solid Manager, Mitzi László gave an overview of Solid as well as a mid-year recap. This was followed by a presentation by Sarven Capadisli on behalf of the Editors to give an overview of the specification progress this month. The two early companies to work on Solid introduced themselves and gave a quick update on what they are currently working on. Kelly O'Brien from Inrupt spoke about the UK covid project with Open University and Kingsley Idehen introduced OpenLink.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-07-02-this-month-in-solid.md:29]:
```suggestion
This month, Solid Manager Mitzi László gave an overview of Solid as well as a mid-year recap. This was followed by a presentation by Sarven Capadisli on behalf of the Editors to give an overview of the specification progress this month. Two early companies to work on Solid introduced themselves and gave a quick update on what they are currently working on. Kelly O'Brien from Inrupt spoke about the UK COVID project with Open University and Kingsley Idehen introduced OpenLink. 
```

File: _posts/newsletter/2020-07-02-this-month-in-solid.md [hunk 2]
```diff
@@ -8,3 +8,44 @@ categories: [Updates]
...
+### Articles and Talks mentioning Solid
+
+* 2020-06-24 BBC Rethink [Tim Berners-Lee: Rethinking Digital Access](https://www.bbc.co.uk/sounds/play/p08ht9tf) 
+
+### In Other News
+
+There's a new category of [conversations on the forum dedicated to local Solid communities](https://forum.solidproject.org/c/local-solid-communities/37). If you would like to connect to others working or interested in Solid in your city of country, open a thread and describe what you are working on so others know where to find you. 
+
+#### Solid Implementations
+* [Solid Groups](https://solid-groups.solid.community/ 22) app with open [source code](https://gitlab.com/angelo-v/solid-groups 7)
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-07-02-this-month-in-solid.md:42]: broken links (numbers)

**Comment by csarven** [comment]
I suggest making the SW video available from solidproject.org eg. `htps://solidproject.org/media/videos/solid-world/2020-07-02.webm` - using whichever URI Template is currently in use.

**Comment by Mitzi-Laszlo** [comment]
All past Solid World recordings are available on https://solidproject.org/events 

There is a problem with storage space at the moment if we store videos on a Pod.

**Pull request merged**